cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/etl/batch/ETLWorkflowTestRun.java

```diff
@@ -217,6 +220,35 @@ public void testBackwardsCompatibleExternalDatasetTrackingSpark() throws Excepti
     testExternalDatasetTracking(Engine.SPARK, true);
   }
 
+  @Test
+  public void testGetPlugins() throws Exception {
```

Comment:
true, but i couldn't find any existing app with plugins available for AppLifecycleHttpHandler test, since this testbase had everything nicely setup, i used it. added a todo to move this to AppLifecycleHandler.